docs/transactions/iti41.md

```diff
@@ -1,35 +1,228 @@
-All non-key elements are ignored by the eMedication service.
+### Rules for each type of document
+#### MTP
+* Treatment duration (defined in [dosage](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-dosage.html)) should be within the period of validity of the document, defined by [`DocumentEntry.serviceStartTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.19) and [`DocumentEntry.serviceStopTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.20) metadata
 
-Some values from the DocumentEntry are checked against the document:
+#### PRE
+* Treatment duration (defined in [dosage](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-dosage-medicationrequest.html) - `CHEMEDEPRDosageMedicationRequest.repeat.bounds`) should be within the [period of validity of the document](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medicationrequest.html) (`CHEMEDEPRMedicationRequest.validityPeriod`).
+
+* Each PRE item should [refer to an MTP item](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-document-medicationprescription.html) (`CHEMEDEPRMedicationRequest.treatmentplan`) that must:
+    * Exist (has already been published, and never deleted)
+    * Be [approved](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.2), ie. `DocumentEntry.availabilityStatus = approved`, 
+    * Be valid, ie. the submission date must be within the [MTP `dosage`'s](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-dosage.html) `boundsPeriod` (`CHEMEDEPRDosage.repeat.bounds`).
+        * If the `boundsPeriod` is not defined, the entry is considered to be always active.
+        * If only a [`startDate`](http://hl7.org/fhir/R4/datatypes-definitions.html#Period.start) is specified, the MTP is considered to be active if the current date is after that date.
+    * Active, ie. `CHEMEDEPRDocumentMedicationTreatmentPlan.CHEMEDEPRMedicationStatement.status = active`
+    * For the same patient.
+ 
+* [PRE document's date of validity](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medicationrequest.html) (`CHEMEDEPRMedicationRequest.validityPeriod` ) must be after the [PRE document's creation time](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-composition-medicationprescription.html) (`CHEMEDCompositionMedicationPrescription.date`) and after the [start date of duration](https://build.fhir.org/ig/hl7ch/ch-emed//StructureDefinition-ch-emed-medicationstatement.html) (`CHEMEDMedicationStatement.effective[x].effectiveDateTime`) of treatment defined in the [MTP referenced by the PRE](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-document-medicationprescription.html) (`CHEMEDEPRMedicationRequest.treatmentplan`).
```

Comment:
The check of the `validityPeriod` against the document creation time is not performed either. This one might make sense but it would have to be discussed to see whether it would be useful at all since in any case document submission should not be blocked by this and it would result in a warning, tops.

The check of the `validityPeriod` against the `effective` field is not implemented and it will never be since it is not supported by our IG and instead the treatment period is defined in the `boundsPeriod` of the base dosage instructions. A check of `validityPeriod` against the treatment period is performed and I don't think it makes much sense to perform such a check (arguably a prescription could be valid even before starting a treatment if the treatment is in the future?).